extension/src/setup/webview/messages.ts

```diff
@@ -28,33 +30,28 @@ export class WebviewMessages {
   }
 
   public sendWebviewMessage({
-    cliCompatible,
-    needsGitInitialized,
     canGitInitialize,
+    cliCompatible,
+    hasData,
+    isPythonExtensionInstalled,
+    isStudioConnected,
     needsGitCommit,
+    needsGitInitialized,
     projectInitialized,
-    isPythonExtensionInstalled,
     pythonBinPath,
-    hasData
-  }: {
-    cliCompatible: boolean | undefined
-    needsGitInitialized: boolean | undefined
-    canGitInitialize: boolean
-    projectInitialized: boolean
-    needsGitCommit: boolean
-    isPythonExtensionInstalled: boolean
-    pythonBinPath: string | undefined
-    hasData: boolean | undefined
-  }) {
+    shareLiveToStudio
+  }: SetupData) {
```

Comment:
Note to self: Consider using "Step" here instead of sending all of this data (consolidate logic).